README_virt.md

```diff
@@ -0,0 +1,92 @@
+
+VIRT Setup instructions
+=======================
+
+`virt` is an `openshift-ansible` provider that uses `libvirt` to create local Fedora VMs that are provisioned exactly the same way that cloud VMs would be provisioned.
```

Comment:
Given this, should we maybe call this "libvirt" instead of just "virt" ?

I can see us adding other virt providers in the future, and "virt" will make this confusing.

So, README_libvirt.md